.circleci/config.yml

```
@@ -1,19 +1,14 @@
-version: 2
-jobs:
-  build:
-    machine: true
-    steps:
-    - checkout
+version: 2.1
 
-    - run: |
-        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-        chmod +x ./architect
-        ./architect version
-    - run: ./architect build
-    - store_test_results:
-        path: /tmp/results
-    - deploy:
-        command: |
-          if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            ./architect deploy
-          fi
+orbs:
+  architect: giantswarm/architect@0.8.3
+
+workflows:
+  test:
+    jobs:
```

We could also add changelog linting job as we go thorough those repos (it was added recently).

```suggestion
    jobs:
      - architect/changelog-lint
```

---

What I applied here does not even seem to be valid. When you point me to some legitimate examples I can incorporate them in my future PRs.